src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let memory = [];
+
+  const adder = (number) => {
+    if (number === undefined) {
+      const sum = memory.reduce((acc, curr) => acc + curr, 0);
```

Comment:
Using `reduce` here is correct, but it's an overkill for summing numbers. Since you're only keeping a total, you could replace the array with a single numeric variable.